resources/compass/values.yaml

```diff
@@ -30,7 +33,25 @@ global:
   gateway:
     host: compass-gateway
     
+  ingress:
```

Comment:
Why there are overrides for ingress present here? Compass should reuse ingress that is already available in kyma.